src/actions/stateManagement.js

```diff
@@ -0,0 +1,7 @@
+import {action, keysToVals} from './util';
+
+export const ACTIONS = keysToVals({
```

Comment:
Nature abhors a vacuum. I abhor string constants. At least the reducers need access to the action types, and I'll be damned if I have to spend 45 seconds debugging a typo.